packages/components/src/text-control/stories/index.tsx

```diff
@@ -17,12 +17,11 @@ const meta: ComponentMeta< typeof TextControl > = {
 	component: TextControl,
 	title: 'Components/TextControl',
 	argTypes: {
-		onChange: {
-			action: 'onChange',
-		},
-		value: {
-			control: { type: null },
-		},
+		help: { control: { type: 'text' } },
+		label: { control: { type: 'text' } },
+		type: { control: { type: 'text' } },
```

Comment:
I wish there was a way to show all the possible input types in a dropdown, rather than allowing any string. But this is definitely a better scenario than the `Object` control

Reply:
Yeah. Ideally I think the `type` prop on `TextControl` should be limited. To me I it doesn't make sense to allow ex `type="button"` and `type="checkbox"` here.

However I think that is a discussion for another PR. If you want I can open a PR and we can continue it there.